scripts/LVGLImage.py

```diff
@@ -1243,6 +1296,8 @@ def convert(self):
             else:
                 img = LVGLImage().from_png(f, self.cf, background=self.background)
                 img.adjust_stride(align=self.align)
+                img.set_rgb565_dither(self.rgb565_dither)
```

Comment:
The PNG is converted to LVGL image in method `from_png`. So the dither parameter should pass directly to it.

```suggestion
                img = LVGLImage().from_png(f, self.cf, background=self.background, rgb565dither= self.rgb565_dither)
                img.adjust_stride(align=self.align)
```

Reply:
OK cool I will make the changes... I will also make the code I added PEP8 compliant.